old-static-site/resources/public/individual.html

```diff
@@ -46,7 +46,9 @@ <h1><a href="/">&nbsp;ClojureBridge&nbsp;MN&nbsp;</a></h1>
           <td class="center-middle" colspan="5">
             You can donate <i>right now</i>!
             <br/>
-            Click below to donate to School Factory (the non-profit that cares for Bridge Foundry and ClojureBridgeMN)
+            Click below to donate
+            to <a href="https://bridgefoundry.org">Bridge Factory</a>.
+            (All donations are provided as unrestricted funds.)
```

Comment:
@tmarble , this file is in `old-static-site`.  Is that directory just for reference?

Reply:
Hmm.. I think that entire directory should be removed

Reply:
Going, going, gone!